lib/graphql/emailDocument.ts

```diff
@@ -0,0 +1,115 @@
+import { GraphQLClient } from "./client";
+import {
+  EmailDocument as EmailDocumentModel,
+  MutationMatchEmailDocumentToTransactionArgs,
+  MutationResult,
+  Query,
+  Transaction as TransactionModel,
+  UserEmailDocumentArgs,
+  UserEmailDocumentsArgs,
+} from "./schema";
+
+type EmailDocumentProps = keyof EmailDocumentModel;
+type TransactionProps = keyof TransactionModel;
+
+const DEFAULT_EMAIL_DOCUMENTS_FIELDS = [
```

Comment:
I added default fields which I think are needed for our webapp usecase.
If we decide later on that we need other fields we can still do it by using optional `fields` and `matchFields`